packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/common_ui_components/app_error_modal_widget.dart

```diff
@@ -40,6 +42,7 @@ class AppErrorModalWidget<BlocType extends RxBlocTypeBase>
                   Navigator.of(context).pop();
                 },
                 onCancelCallback: onCancel,
+                retryButtonText: retryButtonText,
```

Comment:
Can be localise the `retryButtonText` string?

A suggestion:
```
 final String? retryButtonText;
// ...
retryButtonText: retryButtonText ?? context.l10n.nameOfFeature.retryButtonText,

```

Reply:
The Password Reset text passed here is already localized. But, good catch in general. I've added a generic localized "Retry" text to be used here for other features